core/rawdb/database.go

```diff
@@ -659,14 +690,14 @@ func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
 			bodies.Add(size)
 		case bytes.HasPrefix(key, blockReceiptsPrefix) && len(key) == (len(blockReceiptsPrefix)+8+common.HashLength):
 			receipts.Add(size)
+		case IsLegacyTrieNode(key, it.Value()):
```

Comment:
Why move it from 668-669 to 693-694?

Reply:
it is a known issue of inspect db ,  the precondition  of the judgment conditions is used to avoid identifying some header metadata as hash trie data.

Reply:
Got it.